README.md

```diff
@@ -0,0 +1,52 @@
+# Difflint #
+
+Do you have a codebase with inconsistent code style, and you want to
```

Comment:
We have the same intent here; this is more of just a hunch I have about how to communicate most effectively with developers.